packages/vscode-extension/src/devices/AndroidEmulatorDevice.ts

```diff
@@ -622,6 +627,29 @@ async function ensureOldEmulatorProcessExited(avdId: string) {
   }
 }
 
+export async function renameEmulator(avdId: string, newDisplayName: string) {
```

Comment:
and here we actually use "displayName` to name the same parameter we refer to as "deviceName" iin other places?